lib/script-loader.php

```diff
@@ -72,6 +91,41 @@ function gutenberg_enqueue_global_styles() {
 add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_global_styles' );
 add_action( 'wp_footer', 'gutenberg_enqueue_global_styles', 1 );
 
+/**
+ * Adds JS logic to the Customizer preview for live previewing Custom CSS for Block Themes.
+ *
+ * Note: The logic in this function would be back-ported into customize-preview.js.
+ */
+function gutenberg_add_customizer_block_theme_custom_css_preview_js() {
+	if ( ! ( is_customize_preview() && wp_is_block_theme() ) ) {
```

Comment:
Per https://github.com/WordPress/gutenberg/pull/70428/files#r2240947898 this can be just:

```suggestion
	if ( ! wp_is_block_theme() ) {
```